ipaplatform/fedora/constants.py

```diff
@@ -10,6 +10,12 @@
 from __future__ import absolute_import
 
 from ipaplatform.redhat.constants import RedHatConstantsNamespace
+from ipaplatform.osinfo import osinfo
+
+# Fedora 28 and earlier use /etc/sysconfig/nfs
+# Fedora 30 and later use /etc/nfs.conf
+# Fedora 29 has both
+HAS_NFS_CONF = osinfo.id == 'fedora' and int(float(osinfo.version_id)) >= 30
```

Comment:
``osinfo.id == 'fedora'`` is not necessary. The value is in the ``ipaplatform.fedora`` package. You can safely assume that it's always a Fedora-like OS.

I also don't like the ``int(float(osinfo.version_id))`` hack. Let's add proper numeric version tuple to ``ipaplatform.osinfo``, so you can do ``osinfo.version_number >= (30,)``.

```
    @property
    def version_number(self):
        """Version number tuple based on version_id
        """
        version_id = self._info.get('VERSION_ID')
        if not version_id:
            return ()
        return tuple(int(p) for p in version_id.split('.'))
```

Reply:
I'd object to adding another version method. 
int(float(str)) does the job here, or am I missing anything?
As for the other change, will do.

Reply:
I proposed the ``version_number`` function because there is no guarantee how the ``VERSION_ID`` field looks like. The ``float()`` trick is going to fail if e.g. RHEL decides to name the next release ``8.0.0.`` instead of ``8.0``.